pinot-tools/src/main/java/org/apache/pinot/tools/segment/converter/DictionaryToRawIndexConverter.java

```diff
@@ -296,6 +297,10 @@ private void convertOneColumn(IndexSegment segment, String column, File newSegme
       throws IOException {
     DataSource dataSource = segment.getDataSource(column);
     ForwardIndexReader reader = dataSource.getForwardIndex();
+    if (reader == null) {
```

Comment:
same here. Consider renaming it to `forwardIndexReader`.

Reply:
This is a refactoring change (since the original variable was named `reader`). Is it alright if I do this as a follow up to keep this PR simpler? I can check for this in a bunch of other files too then.

Reply:
done